_posts/en/newsletters/2019-10-30-newsletter.md

```diff
@@ -0,0 +1,152 @@
+  0.7.3] adds support for a PostgresSQL backend, makes it possible to
+  send funds directly to a particular address when closing a channel,
+  and allows you keep your HD wallet seed encrypted when `lightningd`
+  isn't running---plus many other features and several bug fixes.
+
+- **Help Bitcoin Core release candidate:** experienced users are encouraged to
+  help test the latest release candidates for the upcoming version of
+  [Bitcoin Core][Bitcoin Core 0.19.0].
+
+## News
+
+- **LN simplified commitments:** in two separate threads, developers
+  of LND discussed their work on implementing simplified commitments,
+  which are LN settlement transactions that only pay a minimal onchain
+  transaction fee and which contain two additional outputs (one for each
+  party).  The idea is to allow either party to choose what transaction
+  fee they want to pay at the time the transaction is broadcast, which
+  they can do using Child-Pays-For-Parent (CPFP) fee bumping from their
+  individual output.  Although this has been discussed in the past, it
+  was vulnerable to an attack that is expected to be resolved by the
+  inclusion of CPFP carve-out in the upcoming Bitcoin Core version
+  0.19.0 (see [Newsletter #56][pr15681]).
+
+    In the first thread, Johan Halseth posted an [email][halseth
+    carve-out] about loosening mempool policy in order to make
+    simplified commitments even simpler.  This received several
+    objections on the basis that a slight change wouldn't be effective
+    and too much change would put the network at risk of
+    bandwidth-wasting attacks.  However, this discussion (and a separate
+    [discussion][rubin justification] started on the #bitcoin-core-dev
+    IRC channel by Jeremy Rubin) revealed that many developers wanted to
+    gain a better understanding of the current rules and how they might
+    be improved.  A good outline of the subject was sourced from a
+    presentation given by Suhas Daftuar that has now been converted into
+    a [wiki page][daftuar duo].
+
+    In the second thread, Joost Jager [resumed][anchor thread] an old
+    thread started by Rusty Russell with a proposed specification for
+    simplified commitments (see [Newsletter #23][opt_simplified]).
+    Based on the upcoming carve-out feature and other developments in
+    LN, Jager makes several suggestions, including: using the name
+    "anchor output" for the outputs meant to be spent with CPFP; using
+    an additional set of pubkeys for the anchors to ease splitting
+    responsibilities between cold and hot wallets; and using static keys
+    to simplify backup recovery.  He subsequently opened a [PR][BOLTs
+    #688] to the BOLTs repository to add simplified commitments to the
+    LN protocol specification.
+
+- **Publication of videos and study material from schnorr/taproot workshop:**
+  Optech published a [blog post][taproot workshop] with links to videos, Jupyter
```

Comment:
nit: sometimes Optech writes "jupyter" and other times "Jupyter"... possibly a candidate for the style guide